docs/user_stories/debug_failed_build.md

```diff
@@ -0,0 +1,23 @@
+### As a developer, with a PR that fails to build, I want to find the root cause so I can fix it and merges
+#### User Journey
+1. GitHub notifies user that build failed
+2. User accesses logs
+3. User looks for a stacktrace with their App’s processes name
```

Comment:
I think 3. and 4. can be compressed into `User locates failure and starts fixing failure"
- We might not have stack traces since it could be linter errors, certificate errors, upload errors, etc...

Reply:
Done